src/protocolgame.cpp

```diff
@@ -204,6 +204,8 @@ void ProtocolGame::login(const std::string& name, uint32_t accountId, OperatingS
 			player->registerCreatureEvent("ExtendedOpcode");
 		}
 
+		player->registerCreatureEvent("NetworkMessage");
```

Comment:
if its auto registered then, what's the point of it being creaturescript?
Move to events.xml